src/app/beer_garden/bg_utils/mongo/__init__.py

```diff
@@ -17,15 +17,17 @@ def setup_database(config):
     from mongoengine import connect, register_connection
     from pymongo.errors import ConnectionFailure, ServerSelectionTimeoutError
 
+    db_config = config.get("db")
+
     try:
         # Set timeouts here to a low value - we don't want to wait 30
         # seconds if there's no database
         conn = connect(
             alias="aliveness",
-            db=config.db.name,
+            db=db_config["name"],
```

Comment:
Is there a specific reason for using dictionary-style lookup here?

Reply:
nope